agent/config/config_test.go

```diff
@@ -373,8 +374,8 @@ func TestInvalidFormatParseEnvVariableDuration(t *testing.T) {
 
 func TestValidForImagesCleanupExclusion(t *testing.T) {
 	defer setTestRegion()()
-	defer setTestEnv("ECS_IMAGE_CLEANUP_EXCLUDE", "amazonlinux:2,amazonlinux:3")()
-	imagesNotDelete := parseImageCleanupExclusionList("ECS_IMAGE_CLEANUP_EXCLUDE")
+	defer setTestEnv("ECS_NONECS_IMAGE_CLEANUP_EXCLUDE", "amazonlinux:2,amazonlinux:3")()
```

Comment:
ECS_NONECS? I'm assuming first ECS is the prefix for internal/ECS env vars?
Howe about ECS_EXCLUDE_CLEANUP_NONECS_IMAGE

Reply:
How about `ECS_EXCLUDE_UNTRACKED_IMAGE`?  I'd like to get rid of "NONECS".

Reply:
yikes, good points. 

> ECS_EXCLUDE_UNTRACKED_IMAGE

i like this, thanks @samuelkarp